charts/trento-server/charts/wanda/templates/NOTES.txt

```diff
@@ -0,0 +1,22 @@
+1. Get the application URL by running these commands:
```

Comment:
I don't know if we need all of this for wanda, but well, I guess it is fine hehe